qlog/src/events/quic.rs

```diff
@@ -63,7 +64,7 @@ pub enum PacketNumberSpace {
 }
 
 #[serde_with::skip_serializing_none]
-#[derive(Clone, Serialize, Deserialize, PartialEq, Eq, Debug)]
+#[derive(Clone, Serialize, Deserialize, PartialEq, Eq, Debug, Default)]
```

Comment:
Open question. For your use case, does default obviate the need for `new()`?

https://github.com/cloudflare/quiche/pull/1931/files#diff-6674de9cf1901c352c7eda222350674ca2c177b89a965d0299d7b5bef8449150R88